homeassistant/components/mobile_app/entity.py

```diff
@@ -0,0 +1,158 @@
+"""A entity class for mobile_app."""
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.const import CONF_WEBHOOK_ID
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import DeviceEntry
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+
+from .const import (ATTR_DEVICE_ID, ATTR_DEVICE_NAME, ATTR_MANUFACTURER,
+                    ATTR_MODEL, ATTR_OS_VERSION, ATTR_SENSOR_ATTRIBUTES,
+                    ATTR_SENSOR_DEVICE_CLASS, ATTR_SENSOR_ICON,
+                    ATTR_SENSOR_NAME, ATTR_SENSOR_STATE, ATTR_SENSOR_TYPE,
+                    ATTR_SENSOR_TYPE_BINARY_SENSOR, ATTR_SENSOR_UNIQUE_ID,
+                    ATTR_SENSOR_UOM, DATA_DEVICES, DOMAIN,
+                    SIGNAL_SENSOR_UPDATE)
+
+
+async def async_setup_mobile_app_entry(sensor_type, hass, config_entry,
+                                       async_add_entities):
+    """Set up the mobile app entity sensor via config entry."""
+    platform_name = "MobileAppSensor"
+
+    if sensor_type == ATTR_SENSOR_TYPE_BINARY_SENSOR:
+        platform_name = "MobileAppBinarySensor"
+
+    platform = globals()[platform_name]
+
+    entities = list()
+
+    webhook_id = config_entry.data[CONF_WEBHOOK_ID]
+
+    for config in hass.data[DOMAIN][sensor_type].values():
+        if config[CONF_WEBHOOK_ID] != webhook_id:
+            continue
+
+        device = hass.data[DOMAIN][DATA_DEVICES][webhook_id]
+
+        entities.append(platform(config, device, config_entry))
+
+    async_add_entities(entities, True)
```

Comment:
Why is this `True`? There is no polling so there won't be any polling.

Reply:
```suggestion
    async_add_entities(entities)
```